src/Database/Languages.php

```diff
@@ -49,9 +50,22 @@ protected function getIndexDefinition(): array
         ];
     }
 
-    public function getByAlpha2(string $alpha2): ?Language
+    public function getByAlpha2(string $alpha2): ?LanguageAlpha2
     {
-        return $this->find('alpha_2', $alpha2);
+        $language = $this->find('alpha_2', $alpha2);
+        if (! $language) {
+            return null;
+        }
+
+        return new LanguageAlpha2(
```

Comment:
Sorry, cant understand your proposal.

i think it's wrong:

```
LanguageAlpha2::getAlpha3()
```

Reply:
entity or value object can't depend from way of search.

Reply:
I will make new pull request later, now is not so much time.